packages/x-tree-view/src/internals/plugins/useTreeViewExpansion/useTreeViewExpansion.types.ts

```diff
@@ -2,10 +2,20 @@ import * as React from 'react';
 import { DefaultizedProps, TreeViewPluginSignature } from '../../models';
 import { UseTreeViewItemsSignature } from '../useTreeViewItems';
 
-export interface UseTreeViewExpansionInstance {
+export interface UseTreeViewExpansionPublicAPI {
```

Comment:
@noraleonte I propose you to switch the logic for the definition of the instance / publicAPI and to instead define the method on `XXXPublicAPI` and use `extend` on `XXXInstance`. 
That way it's easy to be sure that **every** public method has a good JSDoc.

Reply:
That's probably a much better way to do it 👍